bots/coniksbot/coniksbot.go

```diff
@@ -1,3 +1,5 @@
+// Package coniksbot provides the main executable for
```

Comment:
Sure, I understand this. But then the package name makes me confused, so I think it would be better to remove it, any Gopher would know what this is. Let's see what @arlolra  and @Liamsi  say.